maec_package_schema.xsd

```diff
@@ -531,6 +538,85 @@
 			</xs:annotation>
 		</xs:attribute>
 	</xs:complexType>
+	<xs:complexType name="MalwareFamiliyListType">
+		<xs:annotation>
+			<xs:documentation>The MalwareFamilyListType captures a list of malware families relating to the entities in a Package.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element maxOccurs="unbounded" name="Malware_Family"
+				type="maecPackage:MalwareFamilyType">
+				<xs:annotation>
+					<xs:documentation>The Malware_Family field specifies a single malware family in the list.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="MalwareFamilyType">
+		<xs:annotation>
+			<xs:documentation>The MalwareFamilyType captures all of the details pertaining to a Malware Family.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element minOccurs="0" name="Name" type="maecPackage:NameType">
+				<xs:annotation>
+					<xs:documentation>The Name field captures a name of the Malware Family.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element maxOccurs="1" minOccurs="0" name="Aliases"
+				type="maecPackage:MalwareAliasesType">
+				<xs:annotation>
+					<xs:documentation>The Alias field captures a single alias for the Malware Family.</xs:documentation>

```

Comment:
This should be updated to reflect that the field captures multiple malware aliases (since it's of type MalwareAliasesType)